app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -95,7 +95,7 @@ def textual_security_groups
   end
 
   def textual_authentications
-    authentications = @ems.authentication_userid_passwords
+    authentications = @ems.authentications
```

Comment:
This was intentional as it prevents dumping SSH keys and not authentication records.  Perhaps we should come up with a new helper method that handles all the pertinent types... `authentications_for_summary` or something maybe.